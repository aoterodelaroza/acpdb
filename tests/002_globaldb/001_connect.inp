## check: 001_connect.out -a1e-10
## delete: 001_connect.db
## labels: regression quick nodata

## clean up
system rm -f 001_connect.db

## create
disconnect
connect 001_connect.db

## disconnect
disconnect

## connect
connect 001_connect.db

## connect after connect
connect ../zz_source/bsip2-small/bsip2.db

