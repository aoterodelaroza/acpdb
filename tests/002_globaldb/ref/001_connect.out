* SYSTEM: rm -f 001_connect.db

* DISCONNECT: disconnect the current database 

* CONNECT 
Disconnecting previous database (if connected) 
Connecting database file 001_connect.db
Creating skeleton database 

* DISCONNECT: disconnect the current database 

* CONNECT 
Disconnecting previous database (if connected) 
Connecting database file 001_connect.db
Connected database is sane

* CONNECT 
Disconnecting previous database (if connected) 
Connecting database file ../zz_source/bsip2-small/bsip2.db
Connected database is sane

